crates/redstone/src/core/aggregator.rs

```diff
@@ -41,6 +41,7 @@ pub(crate) fn aggregate_values(
     data_packages: Vec<DataPackage>,
     config: &Config,
 ) -> Result<Vec<Value>, Error> {
+    config.verify()?;
```

Comment:
we should verify it at the start, before we decode the payload. (this comment will be outdated when the construction of the config will be changeg)

Reply:
No need to use it anywhere after changes to `Config::try_new() -> Result<Config>` as this will be only possible to create it with `try_new` static method. This will ensure config correctness.